pipelines/broad/rna_seq/BroadInternalRNAWithUMIsPipeline.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+import "RNAWithUMIsPipeline.wdl" as RNAWithUMIsPipeline
+import "../../../tasks/broad/Utilities.wdl" as utils
+
+
+workflow BroadInternalRNAWithUMIsPipeline {
+
+  String pipeline_version = "0.1.0"
+
+  input {
+    # input needs to be either "hg19" or "b38"
```

Comment:
@kcibul we usually use "hg38" in our pipelines to denote the hg38 reference. Do you want to keep the input as "b38" or should we use "hg38" instead?

Reply:
hg38 sounds great, i'll update the inputs/outputs doc